src/base.js

```diff
@@ -1,30 +1,29 @@
-import assign from 'lodash.assign'
-import diff from 'immutablediff'
-import forEach from 'lodash.foreach'
-import isArray from 'lodash.isarray'
-import keys from 'lodash.keys'
-import wrap from 'lodash.wrap'
 import defer from 'lodash.defer'
```

Comment:
Not sure if there's a good alternative? Perhaps rolling our own with setTimeout is good enough in this case?